src/wp-admin/css/site-icon.css

```diff
@@ -75,3 +75,111 @@
 	flex-wrap: wrap;
 	gap: 10px;
 }
+
+.site-icon-preview-inline {
+	display: flex;
+	height: 60px;
+	padding: 8px 0 0 8px;
+	align-items: flex-start;
+	gap: 8px;
+	position: relative;
+	overflow: hidden;
+	box-sizing: border-box;
+}
+
+.site-icon-preview-inline.hidden {
+	display: none;
+}
+
+.site-icon-preview-inline.customizer {
+	width: 275px;
+}
+
+.site-icon-preview-inline.settings {
+	width: 350px;
+	padding: 16px 0 0 16px;
+	gap: 16px;
+	height: 88px;
+	margin-bottom: 20px;
+}
+
+.site-icon-preview-inline.settings .home-icon {
+	width: 58px;
+	height: 58px;
+	border-radius: 12.95px;
+}
+
+.site-icon-preview-inline.settings:after {
```

Comment:
This background is not updated when a new icon is selcted. Use an img-tag or similar instead? And when using solid colors as site icons it is hard to see the icon. Maybe add a opacity to this background, or even skip it. 
And using a black site icon makes the shadow disappear. 

![bild](https://github.com/WordPress/wordpress-develop/assets/11491369/9a9e959c-1bd7-4816-9879-38793d9dcc21)

![bild](https://github.com/WordPress/wordpress-develop/assets/11491369/2d949880-dcb6-490b-92de-f5e3f8086872)


![bild](https://github.com/WordPress/wordpress-develop/assets/11491369/39a4f3a4-5dd5-4a68-a665-7e064ce3c5c1)

![bild](https://github.com/WordPress/wordpress-develop/assets/11491369/d5ec902c-a786-4082-8623-7642e8d5c004)

![bild](https://github.com/WordPress/wordpress-develop/assets/11491369/a50ab40c-da2a-4aa6-89c3-6a10402943d8)